neo/public/javascripts/map/controllers.js

```diff
@@ -183,6 +184,39 @@ angular.module('cloudberry.map', ['leaflet-directive', 'cloudberry.common'])
       };
       $scope.controls.custom.push(info);
 
+      // TODO: change it to the real count sent from the middleware
```

Comment:
then you can just keep that piece of logic locally. otherwise, the reviewers have to spend time on the irrelevant stuff.